.github/workflows/e2eMinVscodeVersion.yml

```diff
@@ -0,0 +1,95 @@
+        description: 'Set the branch to use for automation tests in CJS'
+        required: false
+        default: 'develop'
+        type: string
+      apexE2ETests:
+        description: 'Apex E2E Tests'
+        required: false
+        default: true
+        type: boolean
+      coreE2ETests:
+        description: 'Core E2E Tests'
+        required: false
+        default: true
+        type: boolean
+      deployAndRetrieveE2ETests:
+        description: 'Deploy and Retrieve E2E Tests'
+        required: false
+        default: true
+        type: boolean
+      lspE2ETests:
+        description: 'LSP E2E Tests'
+        required: false
+        default: true
+        type: boolean
+      lwcE2ETests:
+        description: 'LWC E2E Tests'
+        required: false
+        default: true
+        type: boolean
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: true
+        type: string
+
+jobs:
+  Apex_E2E_tests:
+    if: ${{ inputs.apexE2ETests || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/apexE2E.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranchForESM || 'testesm' }}
+      vscodeVersion: '1.86.0'
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
+
+  Core_E2E_tests:
+    if: ${{ inputs.coreE2ETests || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/coreE2E.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranchForESM || 'testesm' }}
+      vscodeVersion: '1.86.0'
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
+
+  DeployAndRetrieve_E2E_tests:
+    if: ${{ inputs.deployAndRetrieveE2ETests || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/deployRetrieveE2E.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranchForCJS || 'develop' }}
+      vscodeVersion: '1.85.2'
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
+
+  LSP_E2E_tests:
+    if: ${{ inputs.lspE2ETests || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/lspE2E.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranchForCJS || 'develop' }}
+      vscodeVersion: '1.85.2'
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
+
+  LWC_E2E_tests:
+    if: ${{ inputs.lwcE2ETests || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/lwcE2E.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranchForCJS || 'develop' }}
+      vscodeVersion: '1.85.2'
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
```

Comment:
Since 1.85.2 is now testing minimum supported version and tested in this workflow, let's have the "Deploy and Retrieve", "LSP", and "LWC" categories be **default unchecked** in e2e.yml so that the same tests aren't run twice.